Rakefile

```diff
@@ -11,12 +11,14 @@ end
 task default: :test
 
 RDoc::Task.new do |t|
+  require "hanna-nouveau"
```

Comment:
This will get required whenever the Rakefile is loaded, so even when running tests via `rake test`. I don't think we need to do that ourselves now that we've added the gem to the Gemfile, RDoc will do it for us.

Reply:
Removed explicit require and it works. Looks like rdoc is smart enough to require missing gems.